vaccine_feed_ingest/runners/us/va_gov/normalize.py

```diff
@@ -7,14 +7,21 @@
 import sys
 from typing import List, Optional
 
-from opening_hours import TimeRange
 from vaccine_feed_ingest_schema import location as schema
 
 from vaccine_feed_ingest.utils.log import getLogger
 from vaccine_feed_ingest.utils.normalize import normalize_url
 
 logger = getLogger(__file__)
 
+try:
+    from opening_hours import Times as TimeRange
+except Exception:
+    logger.warn(
+        "if you see this message in your logs, that means a dependency changed its name in us/va_gov normalizer as predicted and its import needs updating to remove a now-unnecessary check for the old import name."
+    )
+    from opening_hours import TimeRange
+
```

Comment:
@bryanculbertson What do you think of this? as the author of the library, I suspect i will have to rename `Times` to `TimeRange` eventually and this seems like a semi-reasonable precaution so that this program continues to work both before and after the change.

Is this fine or Should i go ahead and change it now?